docs/diagrams/ListEventsSequenceDiagram.puml

```diff
@@ -27,6 +27,8 @@ deactivate ListEventsCommand
 
 AddressBookParser --> LogicManager : u
 deactivate AddressBookParser
+AddressBookParser -[hidden]-> LogicManager
+destroy AddressBookParser
 
 LogicManager -> ListEventsCommand : execute()
 activate ListEventsCommand
```

Comment:
for line 15 and 18 maybe it's better to use `execute(cmd)` instead of `execute("listEvents")` to be consistent with other sequence diagrams

Reply:
yup changed thx